graylog2-server/src/main/java/org/graylog2/utilities/ConfigurationMapConverter.java

```diff
@@ -51,7 +51,12 @@
                     try {
                         value = Integer.parseInt(String.valueOf(entry.getValue()));
                     } catch (NumberFormatException e) {
-                        throw new ValidationException(field, e.getMessage());
+                        // If a numeric field is optional and not provided, use null as value
+                        if ("true".equals(String.valueOf(fieldDescription.get("is_optional")))) {
```

Comment:
This expression could be simplified using [`Boolean#valueOf(String)`](https://docs.oracle.com/javase/8/docs/api/java/lang/Boolean.html#valueOf-java.lang.String-):
```java
if (Boolean.valueOf(fieldDescription.get("is_optional")))
```